src/Tizen.Network.Stc/Tizen.Network.Stc/StcManagerImpl.cs

```diff
@@ -153,16 +153,16 @@ internal Task<IEnumerable<NetworkStatistics>> GetStatisticsAsync(StatisticsFilte
                         _getAllStatsCb_map.Remove(key);
                     }
                 };
-            }
 
-            using (var filterHandle = filter.ConvertToNativeFilter())
-            {
-                int ret = Interop.Stc.GetAllStats(GetSafeHandle(), filterHandle, _getAllStatsCb_map[id], id);
-                if (ret != (int)StcError.None)
-                {
-                    Log.Error(Globals.LogTag, "GetAllStatus() failed , Error - " + (StcError)ret);
-                    throw StcErrorFactory.GetStcException(ret);
-                }
+		using (var filterHandle = filter.ConvertToNativeFilter())
+		{
```

Comment:
fix indent ( replace to space from tab )

Reply:
replaced tab with spaces.